app/controllers/annotations_controller.rb

```diff
@@ -1,9 +1,28 @@
 class AnnotationsController < ApplicationController
-  before_filter :authenticate_user!
+    before_filter :authenticate_user!
 
-  def index
-  end
+    def index
+        loadOptions = {
+            :limit =>       100,
```

Comment:
Unnecessary spacing detected.<br>Use the new Ruby 1.9 hash syntax.<br>Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.